gradle.properties

```diff
@@ -4,15 +4,15 @@ VERSION_NAME=3.0.0
 VERSION_CODE=6
 GROUP=com.treasuredata
 
-POM_URL=www.keen.io
-POM_ORG_NAME=Keen IO
-POM_ORG_URL=http://www.keen.io
+POM_URL=www.treasuredata.com/
+POM_ORG_NAME=Treasure Data
+POM_ORG_URL=https://www.treasuredata.com/
 POM_SCM_URL=https://github.com/treasure-data/KeenClient-java.git
 POM_SCM_CONNECTION=scm:git@github.com:treasure-data/KeenClient-java.git
 POM_SCM_DEV_CONNECTION=scm:git@github.com:treasure-data/KeenClient-java.git
 POM_LICENSE_NAME=The Apache Software License, Version 2.0
 POM_LICENSE_URL=http://www.apache.org/licenses/LICENSE-2.0.txt
 POM_LICENSE_DIST=repo
-POM_DEVELOPER_ID=keenlabs
-POM_DEVELOPER_NAME=Keen IO
+POM_DEVELOPER_ID=treasuredata
```

Comment:
Is it ok to change to TD while we are forking this from Keen Lab ?

Reply:
Nice catch, we should list both developers. I fixed this in [98ddad7](https://github.com/treasure-data/KeenClient-Java/pull/13/commits/98ddad766f3a5ba5ee8a613c75111ca7293bfe21)